buildSrc/src/main/groovy/basebibliotek.alma.integrations.core.java-conventions.gradle

```diff
@@ -58,9 +72,21 @@ tasks.named('build').configure {
     dependsOn tasks.named('shadowJar')
 }
 
+tasks.named('testCodeCoverageReport', JacocoReport) {
+
+    reports {
+        html.required.set(true)
+        xml.required.set(false)
+        csv.required.set(false)
+    }
+}
+
 tasks.named('jacocoTestCoverageVerification', JacocoCoverageVerification) {
     violationRules {
         rule {
+            excludes = [
```

Comment:
Why did you add this? Does it actually fail if this one not present? I think i tried something like this myself some time ago, but it did not have any effect.

I have also tried excluding generated sources from jacoco aggregation but have not found a way to do it yet

Reply:
it had no effect here, done it in other projects, and it worked there....